services/vpc_manager/src/main/resources/application.properties

```diff
@@ -2,6 +2,7 @@ server.port=9001
 #####Microservice url configuration######
 microservices.subnet.service.url=http://localhost:9002
 microservices.route.service.url=http://localhost:9003/
+microservices.gateway.service.url=http://localhost:9015/
```

Comment:
@songxiaoyan We can also add this configuration to the yaml file

https://github.com/futurewei-cloud/alcor/blob/master/kubernetes/services/vpc_manager.yaml